Library/Homebrew/extend/ARGV.rb

```diff
@@ -125,7 +125,7 @@ def git?
   end
 
   def homebrew_developer?
-    include?("--homebrew-developer") || !ENV["HOMEBREW_DEVELOPER"].nil?
+    !ENV["HOMEBREW_DEVELOPER"].nil?
```

Comment:
Yeah we talked about it a bit on the chat; it would be great to move the `ARGV` extension methods in a separate module for all these global settings. The long-term goal is to remove as many references to `ARGV` as possible outside of `cmd/` to have more predictable/testable code (i.e. calling `Homebrew.foo` shouldn’t depend on the arguments on the command-line; if it needs them we should call e.g. `Homebrew.foo(:verbose => true)`) and a clear separation of the core vs. the code that handles what is given on the command-line, so we’ll be able to e.g. automatically generate manpage/completion scripts with all the options, warn if an option is not recognized, etc.